docs/README.md

```diff
@@ -2,7 +2,7 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ## Quick Start
 
-[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/magicbell-io/docs)
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/magicbell-io/public)
```

Comment:
I doubt that this will fix the button, as the docs are now also located in a subdirectory, making gitpod not detect their config file.

Reply:
I'll setup Gitpod for the whole repo, or whatever feels like the best solution, in another PR.